.github/workflows/build_and_release_ecr_image.yml

```diff
@@ -0,0 +1,115 @@
+name: CI & CD
+
+on: [push]
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  lint-and-test:
+    name: 'Lint & Test'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run ESLint
+        run: npx eslint .
+
+      - name: Run Mocha tests
+        run: npx mocha
+  build-tag-push-image:
+    name: 'Build, Tag & Push Image'
+    runs-on: ubuntu-latest
+    env:
+      ECR_REPO: "${{ secrets.PROD_AWS_ACCOUNT_ID }}.dkr.ecr.${{ secrets.AWS_REGION }}.amazonaws.com/mjml"
+      SHORT_COMMIT_ID: ${{ github.sha }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set timestamp with lowercase month
+        run: |
+          echo "TIMESTAMP=$(date +'%Y-%m-%d-%H%M%S')" >> $GITHUB_ENV
+          echo "Timestamp tag is '${{ env.TIMESTAMP }}'"
+      - name: Extract branch name and make docker tag compliant.
+        id: extract_branch
+        run: echo "NORMALIZED_BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | sed -e 's/[^a-zA-Z0-9_.-]/-/g' | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+      - name: Login to core-prod AWS account
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
A bit of future thinking: Direct will want to use this image too, both in prod, UAT and local dev :) Not immediately though, I'd expect we get to our ticket there in 2-3 weeks.

@LBarovic @lceatclub We might need your help with configuring cross-account access to this ECR image 🤔

Reply:
Thank you!!!